Spanner/src/Instance.php

```diff
@@ -510,7 +510,8 @@ public function database($name, array $options = [])
             isset($options['sessionPool']) ? $options['sessionPool'] : null,
             $this->returnInt64AsObject,
             isset($options['database']) ? $options['database'] : [],
-            isset($options['databaseRole']) ? $options['databaseRole'] : null
+            isset($options['databaseRole']) ? $options['databaseRole'] : null,
+            isset($options['mapper']) ? $options['mapper'] : null
```

Comment:
this needs to be documented as a valid entry for `$options` in the PHPDoc for this method